README.md

```diff
@@ -1,5 +1,7 @@
 Example Julia package repo.
 
+[![License: MIT - Permissive License](https://img.shields.io/badge/License-MIT-blue.svg)](https://img.shields.io/github/license/JuliaLang/Example.jl)
```

Comment:
Isn't this redundant with what GitHub shows? I thought one of the point of using a standard license text was that GitHub detected it automatically.

Reply:
It would be.

Reply:
Remove it then?